src/components/table/table-row/table-row.spec.js

```diff
@@ -593,7 +593,7 @@ describe('TableRow', () => {
         expect(cell.props().canDrag).toEqual(true);
 
         assertStyleMatch({
-          cursor: 'move'
+          cursor: 'grab'
```

Comment:
I made this identical change in 4bf6496 in [`FE-1569-react-dnd-and-dls-table-row`](https://github.com/Sage/carbon/commits/FE-1569-react-dnd-and-dls-table-row). Since the change is identical, it should merge cleanly without conflict.